### ISSUE TITLE
"Create" missing method should allow for extensions

### ISSUE THREAD

**Issue by astares** [guillep/pharo-core#11735]
When you follow test driven development (TDD) you can write down your scenario as a test case first and then 
create the note yet implemented methods right from the debugger using "Create" button.

For instance when you have a test like

```Smalltalk
testFooInstance

     self assert: Foo new bar equals: 42
```
the #bar method might not yet be implemented. 

Once you create the method the system asks you to choose the protocol you would like the method to be in. 

![image](https://user-images.githubusercontent.com/5980033/193250194-d204956b-d333-4244-a3a6-9e2a38da23f2.png)

At the moment the dialog presented only allows to select from the existing protocols. But sometimes you
want to create the **method as an extension**. In that case it should allow you to alternatively select a package:

**Comment by MarcusDenker** [comment]
I still think we should just remove that dialog. It is just *horribly* intrusive.

I removed it twice and it got always put back, I never understood why people think this is a good idea.

-> create the method in "as yet unclassified"
-> people can fix that later

At the point when this dialog pops up I *never* am it the state of mind to decide a category. e.g. I do not know what is already there, but most importantly: I do not care at that point.

**Comment by astares** [comment]
We will end up with uncategorized methods then - as one has to apply categorization later. I prefer to directly categorize.
Maybe we can have a setting (influenced by a "dont ask me again" on the dialog) and let the user at first time decide if he wants to categorizes directly or later.

**Comment by Ducasse** [comment]
Yes a setting and the possibility to create a new would be good.
May be we should have a little spec component displaying a list and a button "create".
